roles/matrix-nginx-proxy/defaults/main.yml

```diff
@@ -197,8 +197,8 @@ matrix_nginx_proxy_proxy_matrix_client_api_addr_with_container: "matrix-nginx-pr
 matrix_nginx_proxy_proxy_matrix_client_api_addr_sans_container: "127.0.0.1:12080"
 
 # The addresses where the Matrix Client API is, when using Synapse.
-matrix_nginx_proxy_proxy_synapse_client_api_addr_with_container: "matrix-synapse:8008"
-matrix_nginx_proxy_proxy_synapse_client_api_addr_sans_container: "127.0.0.1:8008"
+matrix_nginx_proxy_proxy_synapse_client_api_addr_with_container: "matrix-synapse:{{ matrix_synapse_container_client_api_port }}"
```

Comment:
We should move this to `group_vars/matrix_servers`. You can define `matrix_nginx_proxy_proxy_synapse_client_api_addr_with_container` as empty here and ensure it's set via `roles/matrix-nginx-proxy/tasks/validate_config.yml `